tests/e2e/lib.sh

```diff
@@ -1252,7 +1253,11 @@ _EO_DETAILS_
         if [[ -f "${STATE_DEPLOYED}" ]]; then
             save_junit_success "${stackrox_deployed[@]}"
         else
-            save_junit_failure "${stackrox_deployed[@]}" "Check the build log"
+            if [[ -f "${QA_DEPLOY_WAIT_INFO}" ]]; then
+                save_junit_failure "${stackrox_deployed[0]}" "Wait for $(cat "${QA_DEPLOY_WAIT_INFO}")" "Check the build log"
+            else
+                save_junit_failure "${stackrox_deployed[@]}"
```

Comment:
Why drop "Check the build log" from this call? 🤔

Reply:
@porridge That was a mistake. Tnx for noticing it!